File: CUSTOR.EICOnline.Angular/src/app/model/tariff.ts
export class TariffModel {
  public TariffId: number;
  public Name: string;
  public NameEnglish: string;
  public IsActive: boolean;
  public AccCode: string;
  public Fee: number;
  public TariffModeId: number;
  public IncrementalFee: number;
  public Quantity: number;
  public selected?: boolean ;

  constructor(tariffId?: number, isActive?: boolean, name?: string, nameEnglish?: string,
              accCode?: string, fee?: number, tariffModeId?: number, incrementalFee?: number, quantity?: number,
              selected?: boolean
  ) {
    this.TariffId = tariffId;
    this.Name = name;
    this.NameEnglish = nameEnglish;
    this.IsActive = isActive;
    this.AccCode = accCode;
    this.Fee = fee;
    this.TariffModeId = tariffModeId;
    this.IncrementalFee = incrementalFee;
    this.Quantity = quantity;
    this.selected = selected;
  }
}

File: CUSTOR.EICOnline.DAL/EntityLayer/Tariff.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace CUSTOR.EICOnline.DAL
{
  public partial class Tariff
  {
    [Key]
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public int? TariffId { get; set; }

    public string Name { get; set; }
    public string NameEnglish { get; set; }
    public string AccCode { get; set; }
    public double Fee { get; set; }
    public double IncrementalFee { get; set; }
    public int TariffModeId { get; set; }
    public int Quantity { get; set; }
    public bool IsActive { get; set; }
  }
}